Makefile

```diff
@@ -75,6 +75,10 @@ install: ## Set up development environment
 	node packages/scripts/vscodesettings.js
 	make build-deps
 
+.PHONY: i18n
+i18n: ## Converts your po files into json to translate volto frontend
+	$(MAKE) -C "./packages/volto/" i18n
+
```

Comment:
This belongs below the `.PHONY: build-deps` target, under the `## Build` section.